### PR TITLE
CHET-188: install psql client on migration stack

### PR DESCRIPTION
**Pull request by theghostwhoforks**
This installs postgres 9.6 client from amazon-linux-extras
Screenshot 2020-03-20 at 15 20 02

### CODE REVIEW

File: templates/migration-stack/pkg/migration-helper.yml.src [hunk 1]
```diff
@@ -105,6 +105,10 @@ Resources:
             commands:
               01_mount:
                 command: /opt/atlassian/dc-migration-assistant/mount-efs.sh
+          install_psql_client:
+            commands:
+              01_install_psql:
+                command: amazon-linux-extras install -y postgresql9.6
```
Comment (nanux) [inline templates/migration-stack/pkg/migration-helper.yml.src]: Should we just install `postgresql11`? I've just tested `amazon-linux-extras install -y postgresql11` and it works fine.
`psql` and `pg_dump` should be backwards compatible with older versions so it should give us more breathing room and higher flexibility for customer deployments.

Comment (theghostwhoforks) [reply templates/migration-stack/pkg/migration-helper.yml.src]: good idea. As long as they are compatible, we can add that in.

I can make the version configurable and default to 11 🤔

File: templates/migration-stack/pkg/migration-helper.yml.src [hunk 2]
```diff
@@ -105,6 +105,10 @@ Resources:
             commands:
               01_mount:
                 command: /opt/atlassian/dc-migration-assistant/mount-efs.sh
+          install_psql_client:
```
Comment (bpar476) [inline templates/migration-stack/pkg/migration-helper.yml.src]: Is there any reason we can't use the cfn-init `packages` property to install psql?

Comment (bpar476) [reply templates/migration-stack/pkg/migration-helper.yml.src]: Looks like `amazon-linux-extras` isn't a supported `cfn-init` package

Comment (theghostwhoforks) [reply templates/migration-stack/pkg/migration-helper.yml.src]: oh yeah! Sorry, I missed this comment last week

File: templates/migration-stack/pkg/migration-helper.yml.src (hunk 1 above)
Comment (theghostwhoforks) [reply templates/migration-stack/pkg/migration-helper.yml.src]: Done

Screenshot 2020-03-30 at 12 21 58

Comment (theghostwhoforks) [reply templates/migration-stack/pkg/migration-helper.yml.src]: The default version is 11

Screenshot 2020-03-30 at 12 23 06

**Comment by bpar476** [comment]
Isn't it still installing postgres 9.6? 🤔

**Comment by theghostwhoforks** [comment]
> Isn't it still installing postgres 9.6? 🤔

Have you seen the screenshot above? With the default value of 11, I've verified that the psql client version is 11.x

**Comment by bpar476** [comment]
> > Isn't it still installing postgres 9.6? 🤔
> 
> Have you seen the screenshot above? With the default value of 11, I've verified that the psql client version is 11.x

Even with `command: amazon-linux-extras install -y postgresql9.6`

**Comment by theghostwhoforks** [comment]
> > > Isn't it still installing postgres 9.6? 🤔
> > 
> > 
> > Have you seen the screenshot above? With the default value of 11, I've verified that the psql client version is 11.x
> 
> Even with `command: amazon-linux-extras install -y postgresql9.6`?

For future reference, commits containing the latest changes weren't pushed to remote. This has been done now

**Pull request merged**